### DISCUSSION TITLE
Is there a way to put an icon the largetitle header?

### DISCUSSION THREAD

**Discussion by dhanushm360** [software-mansion/react-native-screens#931] [Ideas]
Something similar to this:

![image](https://user-images.githubusercontent.com/49541102/117857185-1c7dc900-b252-11eb-8599-385c44957bee.png)

**Comment by tboba** [comment]
Hi @dhanushm360, sorry for the late response 🙏 
Currently you can do similar behavior by using `headerRight` option from screen options. Currently we don't have an implementation of header largetitle icons, which I believe are called 'introspects'. However, it's a great idea to do such thing in the future! Thus, I'm moving this thread to Ideas category.
